bylaws/7-indemnification-dissolution.md

```diff
@@ -22,54 +22,69 @@ The CF Council (CFC) shall assess the Foundation's risks and evaluate whether to
+- [607.0857](http://www.leg.state.fl.us/statutes/index.cfm?App_mode=Display_Statute&Search_String=&URL=0600-0699/0607/Sections/0607.0857.html) Insurance.
 
 ## Indemnification
 
 ### Scope of Indemnification and Standard of Conduct
 
-Subject to the limitations set forth in this Section,
-the Foundation must indemnify directors (councilors), Officers, employees, and agents of the Foundation
-from any action, suit, or proceeding (whether civil, criminal, administrative or investigative)
-arising from that person’s role in the Foundation or service at the request of the Foundation.
+Subject to the limitations set forth in this Section, the Foundation shall, to the fullest extent permitted by applicable law, indemnify and hold harmless any person (a “Covered Person”) who is made a party to, or threatened to be made a party to, any action, suit, or proceeding, whether civil, criminal, administrative, or investigative (other than an action by or in the right of the Foundation) (a “Proceeding”), by reason of the fact that they are or were a council member (acting as a director) or officer of the Foundation, against all liability and loss suffered and expenses (including attorneys' fees) reasonably incurred by that person.
+
+The Foundation may, at the discretion of the CF Council (CFC), extend indemnification to employees, committee members, and other agents of the Foundation as determined appropriate by the CFC.
+
+> [!TIP]
+> This means the Foundation will cover costs and losses for council members and officers who are sued because of their role in the Foundation, as long as they acted honestly and in the Foundation's best interest.
+>
+> The council can choose to extend this coverage to staff and committee members if they decide it’s appropriate.
 
 **Standard of Conduct.**
 This indemnification extends to all reasonable expenses (including attorneys’ fees), judgments, fines, and other amounts paid in settlement by the person in connection with the action, so long as the person:
-(1) acted in good faith and in a manner they reasonably believed to be consistent with the best interests of the Foundation, and
-(2) had no reason to believe their conduct was unlawful, in the case of criminal matters.
+
+1. acted in good faith and in a manner they reasonably believed to be consistent with the best interests of the Foundation, and
+2. had no reason to believe their conduct was unlawful, in the case of criminal matters.
 
 This indemnification does not extend to a person who is found liable to the Foundation in an action brought by the Foundation, or to enforce the Foundation's rights.
 Otherwise, the outcome of an action will not alone create a presumption that a person did not act in good faith, in the best interests of the Foundation, or with lawful intent.
 
+> [!TIP]
+> If the Foundation sues someone or they are found liable to the Foundation, they won't be covered. Just being sued doesn't mean they did something wrong.
```

Comment:
Who is "they" in `they are found liable to the Foundation`? Is "sued" the official term to use?

Reply:
The "TIP" uses less formal/colloquial language. It is an attempt to help folks orient so they can understand what they are reading.